#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_xdialog_man.dpatch by Stan Vasilyev <stan.vasilyev@csun.edu>
##
## DP: Fixes Debian specific paths and spelling errors in Xdialog.man (see bug
## DP: 302661)

@DPATCH@

diff -urNad xdialog-2.2.0~/doc/Xdialog.man xdialog-2.2.0/doc/Xdialog.man
--- xdialog-2.2.0~/doc/Xdialog.man	2006-02-17 22:00:59.000000000 -0800
+++ xdialog-2.2.0/doc/Xdialog.man	2006-02-17 22:10:25.000000000 -0800
@@ -270,9 +270,9 @@
 .br 
 \fI%%mandir%%/man1/Xdialog.1*\fP
 .br 
-\fI%%datadir%%/doc/Xdialog\-%%version%%/*\fP
+\fI%%datadir%%/doc/xdialog/*\fP
 .br 
-\fI%%datadir%%/doc/Xdialog\-%%version%%/samples/*\fP
+\fI%%datadir%%/doc/xdialog/examples/*\fP
 .SH "ENVIRONMENT VARIABLES"
 .LP 
 \fIXDIALOG_HIGH_DIALOG_COMPAT\fR
@@ -306,9 +306,9 @@
 Setting this variable to 1 will prevent the Gdk, GLib and Gtk library from emitting messages via g_log().
 .SH "EXAMPLES"
 .LP 
-Sample scripts are available in the \fI%%datadir%%/doc/Xdialog\-%%version%%/samples/\fR directory.
+Sample scripts are available in the \fI%%datadir%%/doc/xdialog/examples/\fR directory.
 .br 
-Full HTML doc with screen shots, examples and FAQ is available in the \fI%%datadir%%/doc/Xdialog\-%%version%%/\fR directory.
+Full HTML doc with screen shots, examples and FAQ is available in the \fI%%datadir%%/doc/xdialog/html\fR directory.
 .SH "AUTHORS"
 .LP 
 Thierry Godefroy <xdialog@free.fr> (from v1.1.0 onwards)
